components/gly_gen/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import { MetaNode } from '@/spec/metanode'
+import { GeneInfo } from '../service/mygeneinfo'
+import { z } from 'zod'
+
+export const GlyGenGeneType = {
+  gene_name: typeof GeneInfo
+}
+
+export const GlyGenResponse = z.object({
+  queryinfo: z.any(),
+  results: z.array(z.any())
+})
+
+export type GlyGenResponseType = z.infer<typeof GlyGenResponse>
+
+export const GlyGenResponseNode = MetaNode.createData('GlyGenResponse')
+  .meta({
+    label: 'GlyGen Response',
+    description: 'GlyGen response object',
+  })
+  .codec(GlyGenResponse)
+  .view(data => (
+    <>
+      <div><pre>{JSON.parse(data.results)}</pre></div>
```

Comment:
data.results is already parsed by the time it gets here, trying to re-parse it during rendering is the reason this crashes in production. it should actually be:
```
<pre>{JSON.stringify(data.results)}</pre>
```
Going from data to string